### PR TITLE
Material Flow Scenario Request v2.0.0

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

 -->

Closes https://github.com/eclipse-tractusx/sldt-semantic-models/issues/725

## Sample Data

```json
{
  "materialFlowScenarioRequest": {
    "scenarioSimResults": {
      "resultOwnId": "916b5688-8bd8-4d7e-83b9-e0d40939274e",
      "resultOwnSimRunInitial": {
        "owner": {
          "bpns": "BPNS00000007OTZ3"
        },
        "dataQuality": 0,
        "description": "Please simulate asap",
        "comment": "successful simulation ",
        "expirationTimestamp": "2023-03-24T09:15:24.000Z",
        "runId": "0fece48b-c8d1-4180-1a9caca6d67e",
        "shipments": [
          {
            "handlingUnits": [
              {
                "name": "Palette",
                "volume": 1,
                "weight": 189,
                "batches": [
                  {
                    "unitOfMeasurement": "KG",
                    "materialName": "KK1000GR-Gehäuse-Rot",
                    "quantity": 50,
                    "materialNumber": "KK1000GR",
                    "materialHazardousGoods": false,
                    "batchSerialNumber": "Batch_1",
                    "batchOrderId": "Order-0001",
                    "batchExpirationTimestamp": "2023-08-22T16:00:00.000Z",
                    "batchNumber": "45"
                  }
                ],
                "handlingUnitId": "HUT_1",
                "amount": 1
              }
            ],
            "shipmentId": "DE51515151",
            "recipientTimestampPlanned": "2023-04-19T09:00:00.000Z",
            "destination": {
              "bpns": "BPNS00000007OTZ3"
            },
            "recipient": {
              "bpns": "BPNS00000007OTZ3"
            },
            "logistics": {
              "bpns": "BPNS00000007OTZ3"
            },
            "preceding": {
              "bpns": "BPNS00000007OTZ3"
            },
            "splittingAllowed": true,
            "destinationTimestamp": "2023-03-19T09:00:00.000Z"
          }
        ],
        "timestamp": "2023-03-09T14:13:42.806Z"
      },
      "resultOwnSimRunUpdated": {
        "owner": {
          "bpns": "BPNS00000007OTZ3"
        },
        "dataQuality": 0,
        "description": "Please simulate asap",
        "comment": "successful simulation ",
        "expirationTimestamp": "2023-03-24T09:15:24.000Z",
        "runId": "0fece48b-c8d1-4180-1a9caca6d67e",
        "shipments": [
          {
            "handlingUnits": [
              {
                "name": "Palette",
                "volume": 1,
                "weight": 189,
                "batches": [
                  {
                    "unitOfMeasurement": "KG",
                    "materialName": "KK1000GR-Gehäuse-Rot",
                    "quantity": 50,
                    "materialNumber": "KK1000GR",
                    "materialHazardousGoods": false,
                    "batchSerialNumber": "Batch_1",
                    "batchOrderId": "Order-0001",
                    "batchExpirationTimestamp": "2023-08-22T16:00:00.000Z",
                    "batchNumber": "45"
                  }
                ],
                "handlingUnitId": "HUT_1",
                "amount": 1
              }
            ],
            "shipmentId": "DE51515151",
            "recipientTimestampPlanned": "2023-04-19T09:00:00.000Z",
            "destination": {
              "bpns": "BPNS00000007OTZ3"
            },
            "recipient": {
              "bpns": "BPNS00000007OTZ3"
            },
            "logistics": {
              "bpns": "BPNS00000007OTZ3"
            },
            "preceding": {
              "bpns": "BPNS00000007OTZ3"
            },
            "splittingAllowed": true,
            "destinationTimestamp": "2023-03-19T09:00:00.000Z"
          }
        ],
        "timestamp": "2023-03-09T14:13:42.806Z"
      }
    },
    "scenarioParameter": [
      {
        "unitOfMeasurement": "KG",
        "parameterComment": "updated Delivery Date",
        "materialName": "KK1000GR-Gehäuse-Rot",
        "parameterQuantityUpdated": 1,
        "parameterId": "847c71e5-614a-468b-a3a0-674bf2af3004",
        "materialNumber": "KK1000GR",
        "parameterDeliveryDateUpdated": "2023-10-10T09:00:00.000Z",
        "parameterDeliveryDateInitial": "2023-10-09T10:00:00.000Z",
        "parameterOrderId": "OID-011123546",
        "parameterQuantityInitial": 1
      }
    ],
    "scenarioHeader": {
      "scenarioOwnerRole": "Customer",
      "scenarioCreationTimestamp": "2023-10-04T09:10:00.000Z",
      "scenarioExpirationTimestamp": "2023-10-07T09:10:00.000Z",
      "scenarioOwner": {
        "bpns": "BPNS00000007OTZ3"
      },
      "scenarioDescription": "Changes in Delivery Date",
      "scenarioId": "8d464b8b-6977-4952-8a22-0489067ca081",
      "scenarioTitle": "Delivery Modification"
    }
  }
}
```

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.6.0)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by dominikoeh** [comment]
@agg3fe please review

File: io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+   samm:description "Foreign key to simulation result based on initial parameters"@en ;
+   samm:characteristic :SimResult .
+
+:resultOwnSimRunUpdated a samm:Property ;
+   samm:preferredName "resultOwnSimRunUpdated"@en ;
+   samm:description "Foreign key to simulation result based on updated parameter (scenario)"@en ;
+   samm:characteristic :SimResult .
+
+:ScenarioOwnerCharacteristic a samm:Characteristic ;
+   samm:preferredName "ScenarioOwnerCharacteristic"@en ;
```
Comment (agg3fe) [inline io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl:196]: please also add `description`

File: io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+   samm:description "Characteristic describing the aspect Material Flow Scenario Request"@en ;
+   samm:dataType :MaterialFlowScenarioRequest .
+
+:MaterialFlowScenarioRequest a samm:Entity ;
+   samm:preferredName "Material Flow Scenario Request"@en ;
+   samm:description "The Material Flow Scenario Request contains standardized attributes for exchanging scenario requests and the resulting simulation results between two OSIM Manager Instances"@en ;
+   samm:properties ( :scenarioHeader [ samm:property :scenarioParameter; samm:optional true ] :scenarioSimResults ) .
+
+:scenarioHeader a samm:Property ;
+   samm:preferredName "scenarioHeader"@en ;
```
Comment (agg3fe) [inline io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl:52]: preferredName should be human readable and follow normal orthography (e.g., no camel case but normal word separation)
Please correct this in other places also in model.

File: io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl (hunk 1 above)
Comment (dominikoeh) [reply io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl:196]: Added a description for the characteristic

File: io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl (hunk 2 above)
Comment (dominikoeh) [reply io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl]: Updated preferred names

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**